src/schemaText/schema-text-zh.json

```diff
@@ -1365,6 +1365,10 @@
     "mqtt_keepalive_check_interval": {
       "label": "保活检查间隔",
       "desc": "检查传入 MQTT 数据包的频率决定了服务器检查新 MQTT 数据包的频率。<br />如果客户端在一定时间内没有发送任何数据包，这段时间将会累积。<br />一旦累积时间超过 `keepalive-interval * keepalive-multiplier`，连接将会被终止。<br />默认值设置为 30 秒，最小值为 1 秒，最大值为 `keepalive-interval / 2`。"
+    },
+    "mqtt_shared_subscription_initial_sticky_pick": {
+      "label": "共享订阅初始 Sticky 选择策略",
```

Comment:
```suggestion
      "label": "Sticky 共享订阅首选接收者",
```

Reply:
more better?

Reply:
I'm not so sure these two have the same meaning.

Reply:
From the perspective of desc, I think this is what it means. 🤔